build.rs

```diff
@@ -1,23 +1,52 @@
+extern crate cmake;
 #[cfg(not(feature = "nolibsnark"))]
 extern crate gcc;
 
 fn main() {
     #[cfg(not(feature = "nolibsnark"))]
     {
+        let libsnark = cmake::Config::new("/root/libsnark")
```

Comment:
I will also not work on the CI pipeline until this is fixed ;)

Reply:
Yes, in progress here #67